tests/cli/rfhub_importer.py

```diff
@@ -0,0 +1,296 @@
+                          Path(robot.libraries.__file__).parent / 'Collections.py',
+                          Path(robot.libraries.__file__).parent / 'DateTime.py',
+                          Path(robot.libraries.__file__).parent / 'Easter.py',
+                          Path(robot.libraries.__file__).parent / 'OperatingSystem.py',
+                          Path(robot.libraries.__file__).parent / 'Process.py',
+                          Path(robot.libraries.__file__).parent / 'Screenshot.py',
+                          Path(robot.libraries.__file__).parent / 'String.py',
+                          Path(robot.libraries.__file__).parent / 'Telnet.py',
+                          Path(robot.libraries.__file__).parent / 'XML.py'}
+
+
+class RfhubImporterTests(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.fixture_path = FIXTURE_PATH
+        self.client = Client('http://localhost:8000', 'rfhub', 'rfhub')
+        self.rfhub_importer = RfhubImporter((self.fixture_path, ), True, self.client)
+
+    def test_delete_collections(self):
+        with responses.RequestsMock() as rsps:
+            rsps.add(responses.GET, f'{self.client.api_url}/collections/', json={},
+                     status=200, adding_headers={"Content-Type": "application/json"})
+            rsps.add(responses.POST, f'{self.client.api_url}/collections/',
+                     json={'name': 'healtcheck_collection', 'id': 1},
+                     status=201, adding_headers={"Content-Type": "application/json", "accept": "application/json"})
+            rsps.add(responses.DELETE, f'{self.client.api_url}/collections/1/', status=204,
+                     adding_headers={"Content-Type": "application/json", "accept": "application/json"})
+            rsps.add(responses.GET, f'{self.client.api_url}/collections/', json=[{'id': 2}, {'id': 66}],
+                     status=200, adding_headers={"Content-Type": "application/json"})
+            rsps.add(responses.DELETE, f'{self.client.api_url}/collections/2/', status=204,
+                     adding_headers={"Content-Type": "application/json", "accept": "application/json"})
+            rsps.add(responses.DELETE, f'{self.client.api_url}/collections/66/', status=204,
+                     adding_headers={"Content-Type": "application/json", "accept": "application/json"})
+            result = self.rfhub_importer.delete_collections()
+            self.assertEqual({2, 66}, result)
+
+    def test_import_libraries(self):
+        self.rfhub_importer = RfhubImporter((self.fixture_path / 'SingleClassLib',
+                                             self.fixture_path / 'LibWithInit'), True, self.client)
+        with responses.RequestsMock() as rsps:
+            rsps.add(responses.GET, f'{self.client.api_url}/collections/', json={},
+                     status=200, adding_headers={"Content-Type": "application/json"})
+            rsps.add(responses.POST, f'{self.client.api_url}/collections/',

```

Comment:
If you want you could make some helper method for adding this mocked json responses to make the code more concise. But it can be also done later, as different PR.

Reply:
That's exactly what I thought. Should helper method return context manager?